validation/validators.go

```diff
@@ -23,6 +23,16 @@ import (
 	"unicode/utf8"
 )
 
+// CanSkipFuncs will skip valid if RequiredFirst is true and the struct field's value is empty
+var CanSkipFuncs = map[string]struct{}{
+	"Email":   struct{}{},
+	"IP":      struct{}{},
+	"Mobile":  struct{}{},
+	"Tel":     struct{}{},
+	"Phone":   struct{}{},
+	"ZipCode": struct{}{},
+}
```

Comment:
@miraclesu why did you limit it to only those templates? I think any validation should be skipped if the field is empty and not required. I am having a use case with `MinSize` which is not handled here.

Reply:
never mind, I just added them since it is a public var